pkgs/os-specific/linux/lvm2/default.nix

```
@@ -1,48 +1,71 @@
-{ stdenv, fetchgit, fetchpatch, pkgconfig, systemd, udev, utillinux, libuuid
+{ stdenv
+, fetchpatch
+, fetchurl
+, pkgconfig
+, utillinux
+, libuuid
 , thin-provisioning-tools, libaio
-, enable_dmeventd ? false }:
+, enable_cmdlib ? false
+, enable_dmeventd ? false
```

camelCase seems to be the way to go for enable options:

```suggestion
, enableCmdlib ? false
, enableDmeventd ? false
```

---

okay, will update.

---

At least `enable_dmeventd` has already existed before that, so I'll add a commit on top of that and not bother with the rebase.